includes/Checker/Abstract_Check_Runner.php

```diff
@@ -311,7 +311,7 @@ protected function get_checks_instance() {
 		}
 
 		$plugin_basename = $this->get_plugin_basename();
```

Comment:
@mukeshpanchal27 we can remove this line also as we no longer need the plugin basename here